cmd/utils/flags.go

```diff
@@ -509,7 +704,12 @@ func SetupLogger(ctx *cli.Context) {
 	glog.SetV(ctx.GlobalInt(VerbosityFlag.Name))
 	glog.CopyStandardLogTo("INFO")
 	glog.SetToStderr(true)
-	glog.SetLogDir(ctx.GlobalString(LogFileFlag.Name))
+	if ctx.GlobalIsSet(LogFileFlag.Name) {
+		logger.New("", ctx.GlobalString(LogFileFlag.Name), ctx.GlobalInt(VerbosityFlag.Name))
```

Comment:
Though that would touch the logger package too, as there's a lot of relics in there too. Maybe it would be more advisable to do this in a follow up PR so that we don't touch *every* package in this one? :D